semantic/monotype.go

```diff
@@ -366,6 +366,9 @@ func (p *RowProperty) TypeOf() (MonoType, error) {
 
 // String returns a string representation of this monotype.
 func (mt MonoType) String() string {
+	if mt.tbl == nil {
+		return "<monotype: nil>"
+	}
```

Comment:
Is there a reason not to just return the empty string?

Reply:
This was from the changes that @jsternberg suggested.  I like that it makes it explicit that there's no type there, and not some other problem.